### ISSUE TITLE
(macOS) Border.CornerRadius doesn't update when set

### ISSUE THREAD

**Issue by mat1jaczyyy** [AvaloniaUI/Avalonia#2418]
But, changing any other element (such as adding a new child to StackPanel), causes the Border to refresh and redraw.

**Comment by mat1jaczyyy** [comment]
Is there any 'Redraw' method I can call to redraw this immediately, working around the issue temporarily?

**Comment by kekekeks** [comment]
>Is there any 'Redraw' method I can call to redraw this immediately, working around the issue temporarily?

`InvalidateVisual`

**Comment by kekekeks** [comment]
CornerRadius is marked as AffectsRender though. https://github.com/AvaloniaUI/Avalonia/blob/fcb91787ae2e3e4aa81a099a33a85466222918ac/src/Avalonia.Controls/Border.cs#L46

Not sure what is broken here.

**Comment by mat1jaczyyy** [comment]
Here's where I'm hitting this issue in my project:

https://github.com/mat1jaczyyy/apollo-studio/blob/33acd281b3a8052fd57aee9736e4344368256aa5/Apollo/DeviceViewers/GroupViewer.cs#L56

If I hit this line of code (uncommented, of course), the Border Radius does not update on the screen. It updates after I do some other action that adds/removes a control from the Window.

**Comment by mat1jaczyyy** [comment]
Nevermind... I had a mistake in my code which I fixed. The BorderRadius does indeed update the screen. The issue is invalid, sorry for the confusion.

**Comment by mat1jaczyyy** [comment]
Turns out I wasn't crazy after all. This regression happens on macOS. I did further testing on Windows where the issue does not reproduce. I have not tested any other OS environments.

However, I've noticed the BorderRadius does redraw itself when an animation plays inside the UserControl (note I'm setting the BorderRadius of a parent of my UserControl from my UserControl). https://github.com/mat1jaczyyy/apollo-studio/blob/fa555ef52afe65e7d9e433bd73cdb617ec06bbe5/Apollo/DeviceViewers/GroupViewer.cs#L56
